docs/appendices/release-notes/unreleased.rst

```diff
@@ -94,3 +94,5 @@ Fixes
 
 - Fixed a regression introduced in CrateDB ``4.0`` preventing the global setting
   ``cluster.info.update.interval`` to be changed.
+
+- Fixed handling of spaces in `$CRATE_HOME`.
```

Comment:
We usually try to include a symptom that users may have seen when running into the fixed issue. 

```suggestion
- Fixed handling of spaces in `$CRATE_HOME`. Users would get a `No such file or directory` error if the path set via `$CRATE_HOME` contained spaces.
```

Reply:
thx!